lua/README.md

```diff
@@ -0,0 +1,11 @@
+# mLRS - lua #
```

Comment:
the inofficial docs style guide is to write things with their correct way, here it should be Lua, nothing else :)
dito few times below
:)